lispusers/MANAGER

```diff
@@ -825,50 +870,50 @@ Will load DataBaseFNS if necessary" (SUBITEMS ("All" 'DB
                                              (FILESLOAD 'DATABASEFNS)
                                              (CL:FORMAT T 
                                        "~&Global DataBaseFNS Flags:~%%SAVEDBFLG = ~a, LOADDBFLG = ~a"
-                                                    SAVEDBFLG LOADBFLG))
+                                                    SAVEDBFLG LOADDBFLG))
                                           (DBFILE 
                                              (FILESLOAD 'DATABASEFNS)
```

Comment:
not sure if this is worth fixing, but FILESLOAD is a NLAMBDA Interlisp function so that (FILESLOAD 'DATABASEFNS) isn't the preferred usage -- you can use
(FILESLOAD DATABASEFNS)

PF* FILESLOAD

Reply:
This was not changed. It's not worth changing at this time, as you said.